tex/introduction.tex

```diff
@@ -15,7 +15,54 @@ \section{Introduction}
+
+Besides Jupyter, there are other popular implementations of computational notebooks~\citep{Hinsen2019a} that increasingly provide a browser-based way of working with them\footnote{See, e.g., the Matlab Live Editor at https://de.mathworks.com/products/matlab/live-editor.html.}.
+
+The following are user stories that may illustrate what people who end up working via a JuptyerHub on an HPC centre may have in mind:
+\begin{itemize}
+  \item I want to produce a computational notebook that presents my scientific data analysis.
+  All or some of my data are so big, that the processing needs substantial storage and computational resources.
+  \item I want to explore / analyze data that are on an HPC centre and that should not\footnote{e.g., forr reproducibility reasons}, or can not\footnote{e.g. because of volume} be moved to my computer.
+  For the exploration I need to try out and quickly adapt different ideas that I would like to document keeping code and results / figures closely together.
+  \item I am used to working with a specific notebook software and format that I mostly use on my personal computer.
+  Now I want to bring my expertise into a collaboration that runs large simulations on an HPC centre and I would like to keep working with my familiar software without having to learn much about the Unix command line etc.
+  \item I want to do data analysis from a personal device that I cannot use for the task directly, e.g., because I am not allowed to install software myself.
+  \item My device has a modern web browser installed that I can use to browse the internet.
+\end{itemize}
+
+
+\subsection{Resulting technical requirements}
+
+The above user requirements can be broken down into different aspects all or most of which can be solved with a JupyterHub but which should be understood separately:
+\begin{itemize}
+  \item \textbf{Remote access:}  Users want to operate a personal computer and have a way of starting execution on an HPC machine, capturing the results to be displayed on their personal computer.
+  \item \textbf{Minimal software requirements on the user side:}  The computers users can access directly are very diverse.  Most of them, however, do have a web browser.
+  \item \textbf{Web based software on the server side:}  Most computational notebook tools provide a built-in\footnote{For others there may be a way of using, e.g., https://novnc.com/info.html.} way of directly connecting to them via HTTP/HTTPS.
+  \item \textbf{Ability to chose arbitrary resources:}  The resource requirements range from very low memory and CPU demands that could easily be fulfilled on a shared machine to requiring multiple HPC nodes or special hardware like GPUs.
+\end{itemize}
```

Comment:
These are very important aspects! I would, however, move these to the discussion section? It might be overwhelming for a reader to be already confronted with too much detail about what might be the actual base problem that needs to be solved. Maybe, these aspects are also rather suited for the outlook section? Or mention them in the introduction, but only briefly and with relation to later sections?